src/components/Pagination/PaginationButton/PaginationButton.tsx

```diff
@@ -0,0 +1,42 @@
+import { FC } from "react";
+import { useLanguage } from "@/context/LocaleProvider";
+import { PaginationButtonProps } from "./types";
+import { StyledButton, StyledIconWrapper, StyledText, StyledWrapper } from "./styles";
+
+const PaginationButton: FC<PaginationButtonProps> = ({
+  intent,
+  onLoadNext,
```

Comment:
I don't think we need two separate functions, just onClick will be sufficient here

Reply:
correct